react.js

```diff
@@ -10,6 +10,12 @@ module.exports = {
     ],
     plugins: ['react-hooks'],
     rules: {
+        quotes: [
+            'warn',
+            'single',
+            { avoidEscape: true, allowTemplateLiterals: true },
+        ],
+        'comma-dangle': ['error', 'always'],
```

Comment:
i don't think these are needed here, because they are already defined in base.js, which this extends